src/applications/disability-benefits/new-686/config/form.js

```diff
@@ -30,6 +23,17 @@ const formConfig = {
   title: 'New 686',
   defaultDefinitions: {},
   chapters: {
+    chapter1: {
```

Comment:
I'd recommend more semantic naming of chapters and pages.

Reply:
I totally agree, @micahchiang and I had discussed changing this to be the names of the chapter like `veteran-information` so I think I will go with that. Thanks!